docs/docusaurus/versioned_docs/version-0.18/cloud/connect/connect_python.md

```diff
@@ -74,7 +74,7 @@ Environment variables securely store your GX Cloud access credentials.
 
 - Run the following Python code to create a Data Context object:
 
-    ```python name="version-0.18 tutorials/quickstart/quickstart.py get_context"
+    ```python name="tutorials/quickstart/quickstart.py get_context"
```

Comment:
These names no longer need to be updated, because we automatically namespace snippets by version now. See docs/docusaurus/versioned_docs/version-0.18/snippets/quickstart.py in this PR for the corresponding snippet name update!